Libbna/CUSTOM-CMS
# #229: styled login and regi btn

**Testing Steps:**

- Click anywhere on the login or register button and check if it redirects you to the page.
